backend/sample.env

```
@@ -165,3 +165,6 @@ REMOTE_PROMPT_STUDIO_FILE_PATH="unstract/prompt-studio-data"
 
 # Storage Provider for Tool registry
 TOOL_REGISTRY_STORAGE_CREDENTIALS='{"provider":"local"}'
+
+# Highlight data to be available in api deployment
+ENABLE_HIGHLIGHT_API_DEPLOYMENT=True
```

```suggestion
ENABLE_HIGHLIGHT_API_DEPLOYMENT=False
```
Shouldn't this be `False` to retain the existing behaviour of removing the highlight data by default?